src/charm.py

```diff
@@ -67,8 +73,29 @@ def _reconcile(self):
         charm_root = self.charm_dir.absolute()
         forward_alert_rules = cast(bool, self.config["forward_alert_rules"])
 
+        # Dashboards setup
```

Comment:
Would it make sense to extract this to a free function? E.g.

```python3
def forward_dashboards(charm: CharmBase, from_rel: str, to_rel: str):
```
So the in the reconciler we'd just:
```python3
forward_dashboards(
    self,
    "grafana-dashboards-consumer",
    "grafana-dashboards-provider",
)
```